### PR TITLE
refactor(app-wrapper): add deprecation warning (FE-3649)

### PR DESCRIPTION
**Pull request by Parsium**
### Proposed behaviour

- Remove any remaining references of `AppWrapper`  in other components and storybook docs, and replace with `Box`.
- Add deprecation warning to `AppWrapper`'s code and story file

### Current behaviour

`AppWrapper` has no deprecation warning in its code despite plans to remove it. It's also used in a couple of stories.

### Checklist

- [x] Commits follow our style guide
- [ ] Related issues linked in commit messages if required
- [ ] Screenshots are included in the PR if useful
- [ ] All themes are supported if required
- [ ] Unit tests added or updated if required
- [ ] Cypress automation tests added or updated if required
- [x] Storybook added or updated if required
- [ ] Translations added or updated (including creating or amending translation keys table in storybook) if required
- [ ] Typescript `d.ts` file added or updated if required

#### QA

- [ ] Tested in CodeSandbox/storybook
- [ ] Add new Cypress test coverage if required
- [ ] Carbon implementation matches Design System/designs
- [ ] UI Tests GitHub check reviewed if required

### Testing instructions

Check for any major regressions in layout in stories which will now use `Box` instead of `AppWrapper`.

### CODE REVIEW

File: src/components/form/form.stories.mdx [hunk 1]
```diff
@@ -380,7 +380,7 @@ It is possible to render `Form` as a content of `DialogFullScreen` component.
             title="Form in Dialog"
             subtitle="Subtitle"
           >
-            <AppWrapper>
+            <Box p={"0px 40px"}>
```
Comment (samtjo) [inline src/components/form/form.stories.mdx:383]:
```suggestion
            <Box p="0px 40px">
```

File: src/components/dialog-full-screen/dialog-full-screen.stories.mdx [hunk 2]
```diff
@@ -811,7 +810,7 @@ to have a possibility to manage active `Tabs` group
             title="Title"
             subtitle="Subtitle"
           >
-            <AppWrapper>
+            <Box p={"0px 40px"}>
```
Comment (samtjo) [inline src/components/dialog-full-screen/dialog-full-screen.stories.mdx:813]:
```suggestion
            <Box p="0px 40px">
```

File: src/components/app-wrapper/app-wrapper.stories.mdx [hunk 3]
```diff
@@ -1,9 +1,20 @@
...
 import { Meta, Story, Canvas, ArgsTable } from "@storybook/addon-docs";
 import AppWrapper from ".";
+import Box from "../box";
+import DeprecationWarning from "../../__internal__/DeprecationWarning";
+import LinkTo from "@storybook/addon-links/react";
 
 <Meta title="AppWrapper" parameters={{ info: { disable: true } }} />
 
+<DeprecationWarning>
+  AppWrapper is now deprecated. Please use the{" "}
```
Comment (samtjo) [inline src/components/app-wrapper/app-wrapper.stories.mdx:11]:
```suggestion
  AppWrapper is now deprecated. Please use the
```

File: src/components/app-wrapper/app-wrapper.stories.mdx [hunk 4]
```diff
@@ -1,9 +1,20 @@
...
+  <LinkTo kind="Box" story="spacing">
+    Box
+  </LinkTo>{" "}
```
Comment (samtjo) [inline src/components/app-wrapper/app-wrapper.stories.mdx:14]:
```suggestion
  </LinkTo>
```

**Review by samtjo — CHANGES_REQUESTED** [review]
Hi @Parsium - this isn't actually a breaking change as you're just adding the deprecation warning, not deprecating the component. 

So this PR needs to change descriptions etc to be "Add deprecation warning to AppWrapper", or you need to actually remove the component if that's what's intended here 👍🏻

File: src/components/row/row.stories.mdx [hunk 5]
```diff
@@ -10,7 +10,7 @@ import LinkTo from "@storybook/addon-links/react";
 <Meta title="Row" parameters={{ info: { disable: true } }} />
 
 <DeprecationWarning>
-  Row and Column are due to be depricated. Please use the{" "}
+  Row and Column are both now deprecated. Please use the{" "}
```
Comment (DipperTheDan) [inline src/components/row/row.stories.mdx:13]: I probably would leave this as is because Kuba is working on the deprecation of Row and Column.

Comment (DipperTheDan) [inline src/components/row/row.stories.mdx:13]: I probably would leave this as it was. Kuba is currently working on the deprecation of Row and Column.

**Review by samtjo — DISMISSED** [review]

**Review by DipperTheDan — DISMISSED** [review]

**Review by samtjo — DISMISSED** [review]

**Review by DipperTheDan — APPROVED** [review]

**Review by DlgSHi — APPROVED** [review]

**Review by samtjo — APPROVED** [review]

**Review by DipperTheDan — APPROVED** [review]

**Review by DlgSHi — APPROVED** [review]

**Pull request merged**